hr_attendance_management/models/hr_employee.py

```diff
@@ -18,8 +18,10 @@ class HrEmployee(models.Model):
     ##########################################################################
     #                                 FIELDS                                 #
     ##########################################################################
-    extra_hours_status = fields.Selection([(True, 'annual'),
-                                           (False, 'continuous')])
+    extra_hours_continuous_cap = fields.Boolean(
+        help="Set this field to true if you do not want to have the employee "
```

Comment:
Based on the variable name, I would assume the opposite, or I am misunderstanding something? To me, setting `extra_hours_continuous_cap` to true would mean that we want to continuously cap the extra hours, no?